README.md

```diff
@@ -187,7 +187,6 @@ These are put in an autogenerated file at ``cfg/sourcemod/pugsetup.cfg``, once t
 - ``sm_pugsetup_start_delay`` (default 5) - How many seconds before the lo3 process should being.
 - ``sm_pugsetup_team_size`` (default 5) - Maximum size of a team when selecting teams.
 - ``sm_pugsetup_time_format`` (default "%Y-%m-%d_%H") - Time format to use when creating demo file names.
-- ``sm_pugsetup_use_aim_map_warmup`` (default 0) - Whether to change map to a random map from sm_pugsetup_maplist_aim_maps during warmup periods. Note that the plugin will ``exec sourcemod/pugsetup/aim_warmup.cfg`` when on an aim_ map.
```

Comment:
You should update the description for sm_pugsetup_use_aim_map_warmup at L176 as well. (to something like "If using aim map warmup, the maplist...")